src/plugins/intel_npu/src/ops/include/intel_npu/ops/flash_attention_tile.hpp

```diff
@@ -0,0 +1,66 @@
+// Copyright (C) 2025 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "openvino/op/op.hpp"
+
+namespace ov::intel_npu::op {
+
+/// \brief Operation for FlashAttention implementation of SDPA
+class FlashAttentionTile : public ::ov::op::Op {
+public:
+    OPENVINO_OP("FlashAttentionTile", "intel_npu", Op);
+
+    struct Config {
+        bool is_head = false;
```

Comment:
would recommend using enum for FATile type,  if you have more than 2 block types, while we are currently using two type, so is_final_tile, or is_tail would be enough.
https://github.com/openvinotoolkit/openvino/pull/33036/changes#diff-1e35b6970cab68dc82ec10d4677ee814e5f6ddbd8cac1865ba2ecc849d9d2484R548